src/pkg/projects/windowsdesktop/pkg/Microsoft.WindowsDesktop.App.pkgproj

```diff
@@ -51,6 +51,9 @@
 
     <!-- We don't have a ref assembly for DirectWriteForwarder -->
     <IgnoredReference Condition="'$(PackageTargetRuntime)' == ''" Include="DirectWriteForwarder" />
+
+    <!-- We want to intentionally exclude the ref assembly for System.Security.Permissions. -->
+    <IgnoredReference Condition="'$(PackageTargetRuntime)' == ''" Include="System.Security.Permissions" />
```

Comment:
Careful here: who's referencing this?  I would have expected that reference to be gone.  We should wait to remove it until it is.

Reply:
I see... when I remove the ignore, the list is:

```
Assembly 'System.Xaml' is missing dependency 'System.Security.Permissions'
Assembly 'System.Configuration.ConfigurationManager' is missing dependency 'System.Security.Permissions'
Assembly 'System.Security.Cryptography.Xml' is missing dependency 'System.Security.Permissions'
Assembly 'WindowsBase' is missing dependency 'System.Security.Permissions'
Assembly 'System.Windows.Forms.Design.Editors' is missing dependency 'System.Security.Permissions'
Assembly 'System.Drawing' is missing dependency 'System.Security.Permissions'
Assembly 'System.DirectoryServices' is missing dependency 'System.Security.Permissions'
```

Reply:
We need to determine why this stuff is pulling it in.  Dangling type-forwards are OK, actual type references are not.

Reply:
We haven't had an update from WPF for a while, and it looks like there are some relevant changes we don't have yet: https://github.com/dotnet/wpf/compare/ea72094ebe0de0da03c591752559739d6fb67f03...master.

Reply:
Ah... `dotnet-wpf-int` isn't producing for 5.0 yet. Filed https://github.com/dotnet/core-setup/issues/7343 to figure out how to get these updated. For now, I'll keep `System.Security.Permissions` so the rest can be mergeable.

Reply:
That's not the only cause for this.  Some of these other dependencies are concerning.  I suspect we'll need to suppress this, since even once everything is fixed WindowsBase will have typeforwards to System.Security.Permission, but we need to examine all the references before suppressing to find out if there is other work to do to safely omit this.

Reply:
Filed https://github.com/dotnet/core-setup/issues/7344 to follow up on this, I'll keep this PR to what seems doable before then. Thanks for the explanation, this makes sense. (I don't know what the best way to check the references is to get this done myself.)